dag/six_subset.py

```diff
@@ -0,0 +1,68 @@
+"""
+This includes a subset of six.
+
+Copyright is included below.
```

Comment:
The first line of this docstring should be changed to reflect why we brought this in and why we didn't want to just import the library. Blaming me and pointing to the previous discussion should be sufficient.

The second line should die because it's obvious/redundant.